ps_customersignin.php

```diff
@@ -52,6 +52,10 @@ public function __construct()
 
     public function getWidgetVariables($hookName, array $configuration)
     {
+        if (!isset($this->context->customer)) {
+            return false;
```

Comment:
```suggestion
            return [];
```

Actually the output for `getWidgetVariables` is an array. Please don't change the output